consensus/types/src/test_utils/builders/testing_attester_slashing_builder.rs

```diff
@@ -88,14 +114,17 @@ impl TestingAttesterSlashingBuilder {
             );
             let message = attestation.data.signing_root(domain);
 
-            for validator_index in validator_indices {
+            for validator_index in indices_to_sign {
```

Comment:
This could be simplified to `attestation.attesting_indices`, but other than that, looks great!

Reply:
Thats what i had in there initially... and produced the failure of ```invalid_attester_slashing_2_invalid``` and ```invalid_attester_slashing_1_invalid``` :( so the above was the solution to that with my meager understanding of whats going on

Reply:
Oh yeah, actually that makes a lot of sense. Those tests are weird (horrible)